metadata/de.binarynoise.AutomaticAdvancedSettingsExpander/en-US/short_description.txt

```diff
@@ -0,0 +1 @@
+Automatically expands the advanced settings in the Settings app by hooking setInitialExpandedChildrenCount in PreferenceGroup.
```

Comment:
Too long. Short description is limited to 80 chars. Maybe cut everything behind "Settings app"?